src/main/java/frc/robot/poseestimator/limelights/VisionConstants.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.poseestimator.limelights;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class VisionConstants {
+
+	public static final String[] LIMELIGHT_NAMES = new String[] {"limelight-front", "limelight-back", "limelight-gb"};
+
+	public static final SmartLimelightsConfig DEFAULT_CONFIG = new SmartLimelightsConfig(
+		"SmartLimelights/",
+		"LimelightsHardware/",
+		Rotation2d.fromDegrees(20), // ! shall be calibrated
+		0.2 // ! shall be calibrated
+	);
+
+	public static final String LIMELIGHT_LOGPATH_PREFIX = "Limelight ";
+
+	public static final String ESTIMATION_LOGPATH_PREFIX = "estimation_";
```

Comment:
that is not the convention of log paths

Reply:
so what is the convention? Shouldn't be there any form of spacing?